dependencies.sh

```diff
@@ -1,11 +1,10 @@
 #!/bin/bash
 set -e
 
-PIP3='pip3 -q'
+PIP3='pip3 install -q -U'
 APT='apt -qq -y'
 
 echo "* Installing python dependencies"
-  sudo $APT install python3-pip python3-setuptools python3-wheel psmisc
-  $PIP3 install protobuf grpcio grpcio-tools requests prometheus_client
+  sudo $APT install python3-pip python3-setuptools python3-wheel psmisc && $PIP3 -r requirements.txt
```

Comment:
Since we already use `set -e` I don't think we need to merge these lines into a single line.

Reply:
see above.